### PR TITLE
Create Category Controller index method

### PR DESCRIPTION
**Pull request by riola**

### CODE REVIEW

File: src/main/java/com/logisticsplus/forum/controllers/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class CategoryController {
+
+    @RequestMapping(value = "/", method = RequestMethod.GET)
+    public String index(Model model) {
```
Comment (riola) [inline src/main/java/com/logisticsplus/forum/controllers/CategoryController.java]: pse e ki bo qito

**Comment by riola** [comment]
:+1:

File: src/main/java/com/logisticsplus/forum/controllers/CategoryController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class CategoryController {
+
+    @RequestMapping(value = "/", method = RequestMethod.GET)
```
Comment (flakron) [inline src/main/java/com/logisticsplus/forum/controllers/CategoryController.java]: je tu kriju konflikt me url-in e HomeController

Comment (riola) [inline src/main/java/com/logisticsplus/forum/controllers/CategoryController.java]: kem tu provu tash pe hi mi analizu ma detalisht

**Pull request closed without merging**